ui/components/ui/textarea/index.scss

```diff
@@ -0,0 +1,27 @@
+@use "design-system";
+
+.textarea {
+  display: block;
+  box-shadow: none;
+  color: design-system.$black;
+
+  // not sure why this mixin isn't working: @include H6;
```

Comment:
Should we resolve this issue before merging?

Reply:
Ah yes, the mixin works in other components so I'm not exactly sure why it's failing to register here. Going to look into this a bit more.

Reply:
See comment below